conf/nginx/error.html

```diff
@@ -57,13 +56,13 @@
     <!-- Jumbotron -->
     <div class="jumbotron">
         <div class="text-center">
-            <img src="/static/images/ds-logo-horiz.png" alt="DesignSafe-CI Temporarily Unavailable">
+            <img src="/static/images/org_logos/NSF_NHERI-DS.svg" alt="DesignSafe-CI Temporarily Unavailable">
         </div>
         <p class="lead">
             DesignSafe-CI is temporarily unavailable due to maintenance.
             We apologize for the inconvenience. Please try again later.
         </p>
-        <a href="javascript:document.location.reload(true);" class="btn btn-default btn-lg text-center"><span class="green">Try This Page Again</span></a>
```

Comment:
**Why?** This had no effect. Extra maintenance. _I'll restore if there is any concern._